openpgp.d.ts

```diff
@@ -620,6 +624,11 @@ interface KeyPair {
 
 export type EllipticCurveName = 'ed25519' | 'curve25519' | 'p256' | 'p384' | 'p521' | 'secp256k1' | 'brainpoolP256r1' | 'brainpoolP384r1' | 'brainpoolP512r1';
 
+interface SubkeyOptions {
+  sign?: boolean;
+  passphrase?: string;
+}
```

Comment:
The other open PR (#1241) coincidentally does something similar, could you revert this for now to avoid merge conflicts?

Reply:
Heyy @twiss 
Yes! Will do